modules/openideal_media/openideal_media.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'OpenideaL Media'
```

Comment:
Where do we enable this module during the update?

Reply:
Here https://github.com/istolar/openideal/pull/54/files#diff-e42e8e09ccc374119c0a3d6217ad49e5cb71fc929a5c6aacedfd9ec490c61c66R6